src/main/java/org/commonjava/util/sidecar/services/ReportService.java

```diff
@@ -119,12 +121,31 @@ public void storeTrackedDownload( JsonObject message )
             return;
         }
         String originalUrl = entryDTO.getOriginUrl() == null ? "" : entryDTO.getOriginUrl();
-        Response response = trackingService.recordArtificat( trackingId, trackingPath, storeKey.getPackageType(),
-                                                             storeKey.getType().name(), storeKey.getName(), originalUrl,
-                                                             entryDTO.getSize(), entryDTO.getMd5(), entryDTO.getSha1(),
-                                                             entryDTO.getSha256() );
-        logger.debug( "Finished consuming folo record seal event for path:{}, trackingId:{}, rep code: {}, body: {}",
-                      trackingPath, trackingId, response.getStatus(), response.getStatusInfo().getReasonPhrase() );
+        boolean exception = false;
+        try
+        {
+            Response response = trackingService.recordArtificat( trackingId, trackingPath, storeKey.getPackageType(),
```

Comment:
Typo on `recordArtificat`.

Reply:
@sswguo sorry, miss this comment, fix on PR: https://github.com/Commonjava/indy-sidecar/pull/83, which I use my original recovered account, please review, thanks.